composeApp/src/commonMain/kotlin/data/repository/EventsRepository.kt

```diff
@@ -1,35 +1,24 @@
 package data.repository
 
-import data.datasource.Failure
-import data.datasource.LocalDataSource
-import data.datasource.RemoteDataSource
-import data.datasource.ResultType
 import domain.Event
-import domain.EventLocation
-import domain.EventType
-import domain.Speaker
-import kotlinx.datetime.DatePeriod
+import domain.EventsRepository
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.IO
+import kotlinx.coroutines.withContext
 import kotlinx.datetime.LocalDate
 
-class EventsRepository(
-    private val dataSource: LocalDataSource,
-    private val service: RemoteDataSource
-) {
-    suspend fun getAllEvent(period: Int): ResultType<List<Event>, Failure> {
-
-        // TODO FALTA DEFINIR,COMPORTAMIENTO LOCAL Y REMOTO
-        return ResultType.Success(emptyList())
+// TODO inject a dispatchers handler
+class EventsRepository : EventsRepository {
```

Comment:
Could you consider not using the same name for the repository class, for example :
in domain layer: EventRepository
in data layer: CommunityEventRepository

Reply:
That's a really good catch!